docs/hmrc.4

```diff
@@ -766,6 +766,15 @@ line.  If it contains the special character ``*''
 .B hypermail
 will display all header lines.
 .TP
+.B show_received_date = boolean_number
+This controls whether the date the message was received is displayed
+at the end of the message text.  If set to
+.B 0
+the received date is not displayed.
+This option is
+.B enabled
```

Comment:
From line 760, you mean to say this option is disabled by default? It should be the case as it is a new feature that may change the output.